rel/plugins/.gitignore

```diff
@@ -0,0 +1,3 @@
+*.*
+!*.exs
+!.gitignore
```

Comment:
Super minor. But would you mind adding a newline at the end of this file? Mike Burns wrote a fantastic summary of why these get marked by GitHub https://robots.thoughtbot.com/no-newline-at-end-of-file